bin/pulsar

```diff
@@ -287,6 +288,9 @@ PULSAR_LOG_APPENDER=${PULSAR_LOG_APPENDER:-"RoutingAppender"}
 PULSAR_LOG_ROOT_LEVEL=${PULSAR_LOG_ROOT_LEVEL:-"info"}
 PULSAR_LOG_LEVEL=${PULSAR_LOG_LEVEL:-"info"}
 PULSAR_ROUTING_APPENDER_DEFAULT=${PULSAR_ROUTING_APPENDER_DEFAULT:-"Console"}
+if [ ! -d "$PULSAR_LOG_DIR" ]; then
+  mkdir -p "$PULSAR_LOG_DIR"
```

Comment:
Here, We should also create log dir if it doesn't exists, not only try create log dir in `pulsar-daemon.sh`,
Such as, If We try to `pulsar-current/bin/pulsar initialize-cluster-metadata`, we would fail.